dynawo/sources/Modeler/DataInterface/DYNDataInterface.h

```diff
@@ -163,6 +175,13 @@ class DataInterface {
    * @returns the service manager
    */
   virtual boost::shared_ptr<ServiceManagerInterface> getServiceManager() const = 0;
+
+  /**
+   * @brief Clone the current data interface
+   *
+   * @returns cloned data interface
```

Comment:
Please add a warning here to mention that the iidm instances are not cloned

Reply:
Done only in implementation